www/pages/working-with-qgis-plugin.md

```diff
@@ -98,6 +98,14 @@ You can create a new Mergin project using the plugin. The plugin offers two opti
 
 	**Note:** There should be only one QGIS project (qgs or qgz file) within the folder (or subfolders).
 
+The plugin allows you to create the project under your own name or an organization you are a member of, provided that you have write access to the name space.

```

Comment:
Write access to an organisation is not enough - the user needs to be 'owner' or 'admin'